code/modules/projectiles/ammunition/ballistic/revolver.dm

```diff
@@ -1,3 +1,22 @@
+// .310 Short (Sakhno-Manni Revolver)
+
+/obj/item/ammo_casing/short310
+	name = ".310 Short bullet casing"
+	desc = "A .310 Short bullet casing. Casing is a bit of a fib, there is no case, its just a block of red powder that's been trimmed down."
```

Comment:
Wouldn't it be better to call these 'cartridges'? I know this is a copy from the normal strika casing, but it seems weird to still be calling them casings when they're caseless.

Reply:
Judging by the description it was an intentional joke

Reply:
I guess :U